### ISSUE TITLE
RaceMenu

### ISSUE THREAD

**Issue by SynopsiHub** [tiltedphoques/Mod-Compatibility#37]
# Mod information

#### Download link
https://www.nexusmods.com/skyrimspecialedition/mods/19080

#### Dependencies

None. 
RECOMMENDED: SKSE for future additions to racemenu (such as CBBE etc).

# Compatibility

Completely compatible, 0 major issues.

# Additional Information

> As with most body/face changes, make sure you utilize "showracemenu" and adjust your character to your liking right before joining the server, then save your game. This will ensure you look exactly how you want and going into the server doesn't change you.

**Comment by NickMillion** [comment]
In my testing I found that bodyslider adjusted values (i.e. specific bones, height, etc.) are not synced.

**Comment by SynopsiHub** [comment]
> In my testing I found that bodyslider adjusted values (i.e. specific bones, height, etc.) are not synced.

Did you make sure to edit your character prior to connecting and save your game? I haven't had any of these issues so far, although I don't think I specifically edited height. The body morphs certainly work.

**Comment by NickMillion** [comment]
Yes, all characters and players created their character presets and did their intro sequences (or in some cases alternate starts) prior to going online.

Edit: In some cases the characters or presets were made prior to installing SkyrimTogether.

To be specific I am referring only to the "Body Scales" portion added by RaceMenu, shown in screenshot below; the vanilla supported morphs all appear to be working normally.
![image](https://user-images.githubusercontent.com/37994255/178121895-cebd305a-64a4-4cc7-9509-e28c24c0a857.png)

**Comment by Kulrathfiik** [comment]
Also, expressions do not sync

**Comment by lBleyl** [comment]
i can confirm, skin colors syncs, but hair colors dont

**Comment by Dragonissa** [comment]
> In my testing I found that bodyslider adjusted values (i.e. specific bones, height, etc.) are not synced.

We cant sync them.
